src/mlpack/methods/pca/decomposition_policies/stochastic_method.hpp

```diff
@@ -271,7 +259,8 @@ class PCAUpdate
   template<typename... Targs>
   void Update(arma::mat& iterate, Targs... fArgs)
   {
-    updatePolicy.Update(iterate, Fargs...);
+    iterate *= -1;
```

Comment:
I guess, I am still just confused about this bit---shouldn't this be unnecessary if we simply negate the results of `Evaluate()` and `EvaluateWithGradient()` and `Gradient()`?  Sorry if I am overlooking something---it's possible I've misunderstood.  It just seems like extra computation here.